src/executor/attach_op_execs_pass.cc

```diff
@@ -159,6 +159,9 @@ class StatefulComputeExExecutor : public OpExecutor {
     op_ctx.run_ctx = rctx;
 #if MXNET_USE_MKLDNN == 1
     InvalidateOutputs(out_array, req);
+    in_array_fallback = CreateDefaultInputs(in_array);
```

Comment:
Can we just declare std::vector<NDArray> in_array_fallback here?

Reply:
we need the fallback arrays to stay in memory or else we segfault.

Reply:
Cannot see why that is the case. But creating a data member is not a good solution to this. I am not very comfortable of intruding a new data member to this class without much justification. If this PR is only for a temporary workaround I am okay to approve. But please create a JIRA ticket to remove this hack later.